docs/user/contacts.md

```diff
@@ -0,0 +1,7 @@
+# Contact Us
+
+Welcome to the Vald project!!
+Vald is an open source project and we would love to hear from you.
+If you are interested in Vald, or you have any question about Vald, please feel free to contact us.
+For example, please join the [Vald Community](https://join.slack.com/t/vald-community/shared_invite/zt-db2ky9o4-R_9p2sVp8xRwztVa8gfnPA) of the slack and make contact. otherwise, please contact us by email at vald@vdaas.org
```

Comment:
I think it is easier to read, what do you think?

```suggestion

We recommend to communicate on slack, but using E-mail is also welcome.
 
Slack: [Vald Community](https://join.slack.com/t/vald-community/shared_invite/zt-db2ky9o4-R_9p2sVp8xRwztVa8gfnPA) 

E-mail: vald@vdaas.org
```

Reply:
ok, I think your opition is better.
I fix it sonn.